docs/src/development/variables/set-variables.md

```diff
@@ -48,13 +73,15 @@ Variables have several Boolean options you can set in the Console or the CLI:
 | Runtime   | `--visible-runtime` | `true`  | Whether the variable is available at runtime. |
 | Build     | `--visible-build`   | `true`  | Whether the variable is available at build time. |
 
-So if you want the `foo` variable to be visible at build time but hidden during runtime, you can set it like this:
+So if you want the `foo` variable to be visible at build time but hidden during runtime,
+you can set it by running the following command:
 
 ```bash
 platform variable:create --level project --name foo --value bar --visible-build true --visible-runtime false
 ```
 
-You can also change the variable options after you create them (except for sensitive values, which can't be set to non-sensitive). For example, to make the `foo` variable visible at runtime and hidden during the build, run this command:
+You can also change the variable options after you create them (except for sensitive values, which can't be set to non-sensitive).
+For example, to make the `foo` variable visible at runtime and hidden during the build, run this command:
```

Comment:
💅 

```suggestion
For example, to make the `foo` variable visible at runtime and hidden during the build, run the following command:
```

Reply:
Again, slightly outside what I was doing, but an easy fix and an improvement.